### PR TITLE
Make sprinkles global

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Extract text sprinkles to global sprinkles since they are reusable 

## Justify why they are needed
Using sprinkles for commonly used properties reduces the output of CSS

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4092** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4092?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/ui/src/theme/media-query.ts [hunk 1]
```diff
@@ -15,6 +15,13 @@ export const mq = Object.fromEntries(
   }),
 )
 
+// Vanilla extract specific breakpoints object
+export const mediaQueries = Object.fromEntries(
```
Comment (Youakeem) [inline packages/ui/src/theme/media-query.ts:19]: We have the `minWidth` breakpoints in `ui/src/theme/media.ts`. I believe they are the same thing.

File: packages/ui/src/theme/sprinkles.css.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { defineProperties, createSprinkles } from '@vanilla-extract/sprinkles'
+import { mediaQueries } from './media-query'
+import { theme } from './theme'
+import { fontSizes } from './typography'
+
+const textColors = {
```
Comment (Youakeem) [inline packages/ui/src/theme/sprinkles.css.ts:6]: I am wondering if we can just spread `...theme.colors`

Comment (guilhermespopolin) [reply packages/ui/src/theme/sprinkles.css.ts:6]: Good question

**Review by guilhermespopolin — APPROVED** [review]
Nice. I assume you gonna to reuse it in another PR or did I missed something? 🤔

File: packages/ui/src/theme/media-query.ts (hunk 1 above)
Comment (gustaveen) [reply packages/ui/src/theme/media-query.ts:19]: Ah, I see that we have two files for handling media queries, let's remove that when we are fully migrated at least. The syntax is a bit different, but maybe we could map over them and transform in to the correct format. 👍

File: packages/ui/src/theme/sprinkles.css.ts (hunk 2 above)
Comment (Youakeem) [reply packages/ui/src/theme/sprinkles.css.ts:6]: I see, makes sense!

**Review by Youakeem — APPROVED** [review]

**Pull request merged**